token/services/selector/sherdlock/service.go

```diff
@@ -17,17 +18,23 @@ import (
 	"github.com/pkg/errors"
 )
 
-const retrySelectionBackoff = 5 * time.Second
-
 type SelectorService struct {
 	managerLazyCache utils.LazyProvider[*token.ManagementService, token.SelectorManager]
 }
 
-func NewService(tokenDBManager *tokendb.Manager, tokenLockDBManager *tokenlockdb.Manager, metricsProvider metrics.Provider) *SelectorService {
+func NewService(tokenDBManager *tokendb.Manager, tokenLockDBManager *tokenlockdb.Manager, cfg driver.ConfigService, metricsProvider metrics.Provider) *SelectorService {
```

Comment:
Please, instead of importing driver.ConfigService, could you define a local interface with just the methods you need?